Library/Homebrew/rubocops/extend/formula_cop.rb

```diff
@@ -0,0 +1,19 @@
+module RuboCop
+  module Cop
+    class FormulaCop < Cop
+      @registry = Cop.registry
+
+      def file_inside_formula_root?
+        return false unless processed_source.path =~ %r{/Taps/homebrew}
+        true
```

Comment:
> Where else are the Formulae located?

In personal/custom Taps, and I think you can also pass Ruby files directly to `brew style`.